src/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
-    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionPrefix>3.0.0-preview</VersionPrefix>
```

Comment:
Why? We have already published 3.0.0 release and go on with next version, so either 3.0.1-preview or 3.1.0-preview .

Reply:
Sounds good

Reply:
```suggestion
    <VersionPrefix>3.1.0-preview</VersionPrefix>
```